tests/conftest.py

```diff
@@ -164,50 +164,20 @@ def xgboost_version(request):
     return request.param
 
 
-@pytest.fixture(
-    scope="module",
-    params=[
-        "1.4",
-        "1.4.1",
-        "1.5",
-        "1.5.0",
-        "1.6",
-        "1.6.0",
-        "1.7",
-        "1.7.0",
-        "1.8",
-        "1.8.0",
-        "1.9",
-        "1.9.0",
-        "1.10",
-        "1.10.0",
-        "1.11",
-        "1.11.0",
-        "1.12",
-        "1.12.0",
-        "1.13",
-        "1.14",
-        "1.14.0",
-        "1.15",
-        "1.15.0",
-        "1.15.2",
-        "2.0",
-        "2.0.0",
-        "2.0.1",
-        "2.1",
-        "2.1.0",
-    ],
-)
-def tf_version(request):
-    return request.param
+@pytest.fixture(scope="module")
+def tf_version(tensorflow_training_version):
```

Comment:
Will there be a ``tensorflow_serving_version``?

Reply:
this PR creates `tensorflow_training_version`, `tensorflow_inference_version`, and `tensorflow_ei_version`. I'll fix all the fixture usage in the tests in a separate PR - I was worried that this PR had too many changes already 😂